builtin/providers/aws/resource_aws_elasticache_replication_group_test.go

```diff
@@ -235,7 +235,7 @@ func TestResourceAWSElastiCacheReplicationGroupIdValidation(t *testing.T) {
 			ErrCount: 1,
 		},
 		{
-			Value:    randomString(17),
+			Value:    randomString(21),
```

Comment:
Errr. I apparently need to eat something, because I was _super sure_ that `if 17 > 20` would also evaluate to true. For some reason.

Disregard this, I'm speaking nonsense.